SPECS/kernel/kernel.spec

```diff
@@ -203,7 +200,7 @@ for MODULE in `find %{buildroot}/lib/modules/%{uname_r} -name *.ko` ; do \
 %define __spec_install_post\
     %{?__debug_package:%{__debug_install_post}}\
     %{__arch_install_post}\
-    %{__os_install_post}\
+    %__os_install_post\
```

Comment:
I am not confident this change is desired based on the changelog. I assume this is something the linter suggested. Do you know if this behaves as expected?

Reply:
Yeah. It looks like build pipelines failed soon after. I reverted the change and re-run the pipelines for both amd64 and arm64. Both succeeded. Ignoring linter output for now.